packages/platform-sdk-eth-server/src/index.ts

```diff
@@ -126,12 +128,24 @@ export const subscribe = async (flags: {
 	server.route({
 		method: "GET",
 		path: "/wallets/{wallet}/transactions",
-		handler: (request) =>
-			database
+		options: {
+			validate: {
+				params: Joi.object({
+					wallet: Joi.string().length(42),
+				}).options({ stripUnknown: true }),
+				query: Joi.object({
+					page: Joi.number().optional().min(1).default(1),
+				}).options({ stripUnknown: true }),
+			},
+		},
+		handler: (request) => {
+			const offset = (request.query.page - 1) * PAGE_SIZE; // TODO Do we start page numbers at 1 or at 0?
```

Comment:
Pages always start at 1. If we would use the offset as a raw value we would start at 0, so this should be correct.